client/src/redux/sagas/pullsTestmanSaga.js

```diff
@@ -0,0 +1,27 @@
+import { takeEvery, call, select, put } from 'redux-saga/effects';
+import { PULLS } from '../constants';
+import { fetchTests } from '../api';
+import { setTestman, setTestmanError } from '../actions';
+
+function* handleTestmanLoad() {
+  try {
+    const pulls = yield select(state => state.pulls);
+    const page = 1;
```

Comment:
I don't see the meaning of the variable

Reply:
I am sending page =1 as testman data might have more than one page (or we can directly send 1 so that we can iterate over page variable in fetchTests() in api/index.js)

Reply:
You plan to use it in future? I mean atm you just assign 1 to it and pass as a parameter